apps/codeforafrica/src/components/NavBarNavList/index.js

```diff
@@ -0,0 +1,72 @@
+import { Link } from "@commons-ui/next";
+import Box from "@mui/material/Box";
+import { useTheme } from "@mui/material/styles";
+import SvgIcon from "@mui/material/SvgIcon";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import PropTypes from "prop-types";
+import React from "react";
+
+import { ReactComponent as TwitterDesktopIcon } from "@/codeforafrica/assets/twitterDesktop.svg";
+import { ReactComponent as TwitterMobileIcon } from "@/codeforafrica/assets/twitterMobile.svg";
+import NavList from "@/codeforafrica/components/NavList";
+import NavListItem from "@/codeforafrica/components/NavListItem";
+
+function NavBarNavList({ menu, direction }) {
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down("sm"));
+
+  if (!menu?.length) {
+    return null;
+  }
+  return (
+    <Box component="nav">
+      <NavList direction={direction} sx={{ alignItems: "flex-start" }}>
```

Comment:
```suggestion
      <NavList direction={direction} sx={{ alignItems: "flex-start" }}>
```
Set nav on `MobileNavBar` and `DesktopNavBar` so that you can use also justify left (mobile) and right (desktop).